app/src/home.js

```diff
@@ -193,7 +205,23 @@ export default function Home() {
   };
 
   const addTranscriptChunk = useCallback(({ Alternatives, IsPartial, StartTime }) => {
-    const text = Alternatives[0].Transcript;
+    let addSpeakerLabel = true;
```

Comment:
I think we refactored this. Might have gotten lost in rebases/conflict resolutions

Reply:
This is based on Shivani's branch and hence it has these changes